src/quicr_server.cpp

```diff
@@ -209,18 +210,33 @@ QuicRServer::TransportDelegate::on_connection_status(
   const qtransport::TransportContextId& context_id,
   const qtransport::TransportStatus status)
 {
+  std::stringstream log_msg;
+  log_msg << "connection_status: cid: " << context_id << " status: " << int(status);
+  server.log_handler.log(qtransport::LogLevel::debug, log_msg.str());
```

Comment:
Feel free to ignore this one, but it's a shame `LogHandler` doesn't have << and >> operators to skip the creation of the stringstream.

Reply:
@GhostofCookie  good idea .. Would you please add those 2 operations on LogHandler in a different PR ?

Reply:
Agree.  Would be great to improve LogHandler.